datahub-graphql-core/src/main/resources/search.graphql

```diff
@@ -1214,6 +1225,11 @@ input BrowseV2Input {
   The search query string
   """
   query: String
+
+  """
+  Optional: Form filter on how to search for entities by form
+  """
+  formFilter: FormFilter
```

Comment:
i think it's safe to just leave anything related to `formFilter` out for now as well since that's only used in the bulk experience. might as well not muddy up these APIs if we're not planning on using it